base/collection_xattr.go

```diff
@@ -191,53 +188,99 @@ func (c *Collection) SubdocWrite(ctx context.Context, k string, subdocKey string
 	return casOut, err
 }
 
-// SubdocGetBodyAndXattr retrieves the document body and xattr in a single LookupIn subdoc operation.  Does not require both to exist.
-func (c *Collection) SubdocGetBodyAndXattr(ctx context.Context, k string, xattrKey string, userXattrKey string, rv interface{}, xv interface{}, uxv interface{}) (cas uint64, err error) {
+// subdocGetBodyAndXattr retrieves the document body and xattrs in a single LookupIn subdoc operation.  Does not require both to exist.
+func (c *Collection) subdocGetBodyAndXattrs(ctx context.Context, k string, xattrKeys []string, fetchBody bool) (rawBody []byte, xattrs map[string][]byte, cas uint64, err error) {
+	xattrKey2 := ""
+	if !c.IsSupported(sgbucket.BucketStoreFeatureMultiXattrSubdocOperations) {
+		if len(xattrKeys) > 2 {
```

Comment:
I think a comment might be merited here to explain that this is backward compatibility for "one system xattr and one user xattr" support.

For that matter, should we return error in this case if xattrKey2 is a system xattr (underscore prefix), to prevent potential misuse?